less/components/modals.less

```diff
@@ -23,6 +23,7 @@
     .box-shadow(@gb-modal-shadow);
 
     &.without-backdrop {
+        z-index: auto;
```

Comment:
[initial-value = auto](https://developer.mozilla.org/en-US/docs/Web/CSS/z-index)
So as you prefer